.gitlab-ci.yml

```diff
@@ -105,6 +105,23 @@ publish-draft-release:
   interruptible:                   true
   allow_failure:                   true
 
+test-deterministic-wasm:
+  stage:                           test
+  <<:                              *docker-env
+  except:
```

Comment:
@gnunicorn  I think you forgot something here, and somewhy it passed the YAML linter without having a value. When do you want to run this job?